packages/@aws-cdk/aws-codepipeline/test/integ.lambda-pipeline.expected.json

```diff
@@ -181,7 +175,13 @@
             ],
             "Name": "Lambda"
           }
-        ]
+        ],
+        "ArtifactStore": {
+          "Location": {
+            "Ref": "PipelineArtifactsBucket22248F97"
+          },
+          "Type": "S3"
+        }
```

Comment:
Same comment here (try to revert this change and see if the compare tests still pass, they should).